drivers/espi/Kconfig.npcx

```diff
@@ -0,0 +1,34 @@
+# NPCX eSPI driver configuration options
+
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config ESPI_NPCX
+	bool "Nuvoton NPCX embedd controller (EC) ESPI driver"
+	depends on SOC_FAMILY_NPCX
+	help
+	  This option enables the Intel Enhanced Serial Peripheral Interface
+	  (eSPI) for NPCX family of processors.
+
+if ESPI_NPCX
+
+config ESPI_OOB_CHANNEL
+	default  y
```

Comment:
I follow what Intel did in https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/espi/Kconfig.xec#L14. I found 'config ESPI_OOB_CHANNEL' doesn't declare 'default' property in https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/espi/Kconfig#L66. Is it still an overriding case?

Reply:
Why providing the test after bool, it makes it overrideable

Oh, you are trying to select the existing Kconfig option. I see. I though you were defining a new options. Can you add a comment saying that we are defaulting the existing option to y since that is the typical use case

Reply:
Done.